src/main/java/sirius/search/Query.java

```diff
@@ -233,6 +234,20 @@ protected Query(Class<E> clazz) {
     }
 
     /**
+     * Adds an <tt>exclude</tt> constraint to the query in the id field.
+     *
+     * This effectively excludes all entities from the given list from the query.
+     *
+     * @param entities the entities to exclude
+     * @return the query itself for fluent method calls
+     * @see NoneInField
+     */
+    public Query<E> exclude(EntityRefList<E> entities) {
```

Comment:
Shouldn't we use a List<String> where (aka entities.getIds()) to make it usable without an entityReflist